modules/apps/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/npm/builtin/BaseBuiltInJSModuleServlet.java

```diff
@@ -114,7 +124,35 @@ private void _sendResource(
 
 		JSPackage jsPackage = resourceDescriptor.getJsPackage();
 
-		URL url = jsPackage.getResourceURL(resourceDescriptor.getPackagePath());
+		URL url = null;
+
+		if (PropsValues.WORK_DIR_OVERRIDE_ENABLED && pathInfo.endsWith(".js")) {
```

Comment:
Thanks for pointing this out.  I will update the deployFast tasks so that they just include everything from the classes folder which should be more correct.

Reply:
This is something we may need to take care of in the future. Now we only serve .js and .map from this servlet, and we don't expect to serve anything else, but who knows. 

So, how would we notice that this is going to fail if we add more file types in the future? Can we do something now to make it easier to detect? :thinking: